templates/splash/api-en.html

```diff
@@ -40,6 +41,8 @@ <h2>Known instances</h2>
 
 <h2>Version history</h2>
 <dl>
+	<dt>Version v2.2 - 2022-10-31</dt>
+	<dd>Addition of the <code>translation</code> attribute set to "no" to each language selection buttons: English and Français</dd>
```

Comment:
Name of the attribute is `translate`:
```suggestion
	<dd>Addition of the <code>translate</code> attribute set to "no" to each language selection buttons: English and Français</dd>
```